javascript-and-json/main.js

```diff
@@ -0,0 +1,30 @@
+const arr = [
+  {
+    isbn: '9781234567897',
+    title: 'Bugs Life Events',
+    author: 'Bug Smith',
+  },
+  {
+    isbn: '97812345677832',
+    title: 'Life is cool',
+    author: 'JT',
+  },
+  {
+    isbn: '82673434567897',
+    title: 'Volcanos suck',
+    author: 'Random',
+  },
+];
+
+console.log(arr);
+console.log('typeof', typeof arr);
+
+const jsonArr = JSON.stringify(arr);
+
+console.log(jsonArr);
+console.log('typeof', typeof jsonArr);
+
+const jsonString = '{"Number id": 456, "name": "Bug Smith"}';
+
```

Comment:
Missing a `JSON.parse(jsonString)` and missing 2 console logs that go with it

Reply:
added to a new var, and console logged it, the JSON string is now showing as an object.

Gratzi!